packages/web/src/scss/tools/_placement.scss

```diff
@@ -102,31 +120,82 @@ $_placement-inverse-map: (
     );
 }
 
-// Function to return an inverse placement name
-// Example: inverse('top') will return 'bottom'
-// Example: inverse('top-start') will return 'bottom'
-// Example: inverse('top-start', true) will return 'bottom-start'
-// Example: inverse('top-start', true, false) will return 'bottom start'
-@function inverse($placement, $main-axis-only: false, $hyphenate: true) {
-    $hyphen: '-';
-    $placement-chunks: string.split($placement, $hyphen);
-    $placement-inverse-chunks: ();
-    $join-with: if($hyphenate, $hyphen, ' ');
-
-    @for $i from 1 through list.length($placement-chunks) {
-        $chunk: list.nth($placement-chunks, $i);
-        $new-chunk: null;
-
-        @if $main-axis-only and $i > 1 {
-            $new-chunk: $chunk;
-        } @else {
-            $new-chunk: map.get($_placement-inverse-map, $chunk);
+// Function to get the cross-axis direction
+// Example: get-cross-axis-direction('left') will return 'vertical'
+// Example: get-cross-axis-direction('bottom') will return 'horizontal'
+@function get-cross-axis-direction($main-axis) {
+    @return if(string.index($main-axis, 'left') or string.index($main-axis, 'right'), 'vertical', 'horizontal');
+}
+
+// Function to check if a placement is logical
+// Example: is-logical('top') will return false
+// Example: is-logical('start') will return true
+@function is-logical($placement) {
+    @return if(string.index($placement, 'start') or string.index($placement, 'end'), true, false);
+}
+
+// Function to return a physical placement name
+// Example: logical-to-physical('start', $direction: 'horizontal') will return 'left'
+@function logical-to-physical($logical, $direction) {
```

Comment:
It is good practice to use a verb in the function name but in this case, I am not quite sure if will be better or not.

Reply:
Added a verb